### DISCUSSION TITLE
'<current_date>.<hash>' as the autogenerated revision id

### DISCUSSION THREAD

**Discussion by VasiliyKirstia** [sqlalchemy/alembic#943] [Ideas]
The problem I have faced using alembic is that it is quite difficult to track changes via autogenerated revisions. For example, to open all files in the order they were created I have to check the current revision in a database, then I have to find the file with the given hash, then I have to find the file with down_revision and so on.

My life would be much easier if all revisions (file names) were prefixed with the creation data.
[Here](https://github.com/VasiliyKirstia/alembic/commit/169fdeb5f75a547f202b9b17ddd80cfaa4fd1563) is my solution.

**Comment by CaselIT** [comment]
Hi,

you can change the file name template quite easily in the configuration: https://alembic.sqlalchemy.org/en/latest/tutorial.html#editing-the-ini-file look for the file template documentation

**Reply by VasiliyKirstia** [reply]
Thanks a lot! It completely solves my problem.
